test/views/cancellation/CancelDeclarationViewSpec.scala

```diff
@@ -53,7 +53,7 @@ class CancelDeclarationViewSpec extends AppViewSpec with CommonMessages {
         )
     )
 
-    checkErrorsSummary(view)
+    view.getElementById("error-summary-heading").text() mustNot be(empty)
```

Comment:
I'm confused.  This change seems quite repetitive.  Why not change `checkErrorSummary` to do the same thing as above?    Or am I missing something?

Reply:
Problem was in way how scala resolved implicits - it resolve earlier `MustMatchers` implicit defs rather then which comes form `ViewMatchers` and cause errors in usage. Hopefully problem was solved.